Task2.js

```diff
@@ -0,0 +1,88 @@
+//=========================2=УРОК=========================//=========================2=УРОК=========================//
+
+// Создание отдельного класса для продукта-обьекта
+class Item {
+    // Функция-конструктор в классе продукта с двумя свойствами и одним методом(функцией)
+    constructor(title, price) {
+        this.title = title;
+        this.price = price;
+    }
+    // Функция-художник в классе продукта (художник нужен при отрисовке продукта)
+    render() {
+        return `<div class="item"><h3 class="item__title">${this.title}</h3><p class="item__price">${this.price}</p></div>`;
+    }
+}
+
+// Cоздание класса корзины, где будут хранится наши продукты после перемещения
+class Catalog {
+    // Функция-конструктор в классе продукта с одним пустым массивом
+    constructor() {
+        this.goods = [];
+    }
+    // Функция для добавления из сервера товаров-продуктов (уже добавлены)
+    fetchGoods() {
+        this.goods = [
+            {},
+            { title: 'Жесткий диск SSD 1Tb', price: 10000 },
+            { title: 'Материнская плата', price: 4000 },
+            { title: 'Видеокарта', price: 15000 },
+        ];
+    }
+    // Функция для отрисовки товаров-продуктов в новой корзине
+    render() {
+        let listHtml = '';
+        this.goods.forEach(good => {
+            good.title = good.title || 0;
+            good.price = good.price || 0;
+            const goodItem = new Item(good.title, good.price);
+            listHtml += goodItem.render();
+        });
+        document.querySelector('.catalog').innerHTML = listHtml;
+    }
+
+    // Функция для подсчета стоимости товаров
+    SumPrices() {
```

Comment:
C заглавной буквы только имена классов
```suggestion
    sumPrices() {
```